src/feature/post/ui/modal/WriteCVModal.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from '@/components/ui/button';
+import { ModalFloatBackground } from '@/components/ui/layout';
+import { useDialog } from '@/shared/modal/hooks';
+import { useRouteNavigation } from '@/shared/route/useRouteNavigation';
+
+export const WriteCVModal = ({
+  onClose,
+  postId,
+}: {
+  onClose: () => void;
+  postId: string;
+}) => {
+  const { isVisible, handleClose: handleCancel } = useDialog({
+    onClose,
+  });
+  const { toPatchProfile, toCreateCoffeeChat } = useRouteNavigation();
+
+  return (
+    <ModalFloatBackground
+      variant="transparent"
+      isVisible={isVisible}
+      onOutSlideClick={handleCancel}
+    >
+      <div className="flex flex-col gap-[14px] text-center">
```

Comment:
tailwind.config.js 보니 rem 관련 정책이 완전히 정리되진 않은 느낌이긴 한데 일단 남겨봅니다

```suggestion
      <div className="flex flex-col gap-3.5 text-center">
```

또는

```suggestion
      <div className="flex flex-col gap-[0.875rem] text-center">
```

Reply:
넵.. 현재는 rem이랑 px를 혼용하여 사용하고 있었고, 이를 rem으로 변경하는 작업이 필요할 것 같습니다. 구체적인 정책 관련해서는 논의 후에 새로 이슈를 파서 작업을 해야 할 것 같고, 아래와 같이 수정할 것을 제안한 상태입니다.

-----
**[px과 rem 관련 정책 제안]**
현제 tailwind를 사용하다보니 px와 rem을 혼용하여 사용하고 있는데, 관련해서 정책을 결정할 필요가 있을 것 같습니다.

**세 줄 요약**
- rem으로 통일
- 대신 일일이 px를 rem으로 변환하는 것은 너무 불편함 -> pxr이라는 단위 도입
- 예시) 16px => 1rem = 16pxr, 10px => 0.625rem = 10pxr, ...

**1. px vs rem, 뭘 사용할지**
먼저 px와 rem의 차이점에 대해 설명 드리도록 하겠습니다.
- px: 말 그대로 화면에서 몇 칸을 차지하는지, 고정된 단위
- rem: html 문서 기본 글꼴 크기에 비해서 얼마나 큰지, 상대적인 단위

이때 퍼블리싱을 진행할 때는 보통 rem을 사용하는 것이 더 유리합니다. 그 이유는
- 만약 기본 브라우저 글꼴 크기가 달라진다면, px로 코딩한 경우에는 사용자가 설정한 기본 글꼴 크기가 변경되지 않습니다.
보통 할머니 할아버지 핸드폰 보면 텍스트 크기가 엄청 크게 설정되어 있죠. 그런 식으로 시력에 따라 글꼴 크기를 키워놨을 수 있는데, 이러한 설정이 px를 사용했을 때에는 무력화됩니다.
- 기기에 따라서 폰트 크기를 변경해야 하는 경우에도 rem이 유리합니다. 컴퓨터에서의 글꼴 크기보다 모바일에서의 글꼴 크기를 모두 일정 비율로 줄이고 싶다거나 할 때 사용할 수 있습니다.
- _(제일 큰 이슈)_ tailwind의 경우 gap-4와 같은 것들이 모두 rem으로 설정되어 있으므로 px를 사용했을 때 기본 글꼴이 달라짐에 따라 스타일링이 원하는대로 나오지 않을 수 있습니다.

**2. tailwind에 어떻게 rem을 도입할까**
일단 현재 rem을 도입하기 어려운 이유는 다음과 같습니다.
- 피그마에서는 px로 적혀 있습니다. 이를 rem으로 바꿔주려면 항상 별도의 계산을 해줘야 합니다.
10px = 0.625rem (기본 글꼴 16px일 때) -> 일일이 계산하기에는 너무 더러운 값입니다. 물론 쓰다보면 익숙해지겠지만..
- 62.5% 꼼수를 넣으면 tailwind에서 제공하는 기본적인 클래스들 (gap-4 등등)의 값이 공식문서와 달라집니다.

그냥 rem으로 계산해도 되기는 하는데요.. 개인적으로 **사람이 계산하기에 직관적으로** 바꾸는 게 작업 시에 더 편할 것 같다는 생각이 들었습니다.
다른 곳에서도 비슷한 어려움을 겪었던 것으로 보이고, 그 중에서 제가 생각하기에 괜찮아 보이는 방법으로는 "새로운 단위 도입"이 있었습니다. ([참고 아티클](https://fe-developers.kakaoent.com/2022/221013-tailwind-and-design-system/))
- 입력은 px과 동일한 값으로 하되, 실제 페이지에서는 rem 단위로 넣어줍니다.
 16px => 1rem = 16pxr

```
const pxToRem = (px, base = 16) => `${px / base}rem`;

module.export = {
  ...
  theme: {
    ...
    spacing: {
      ...range(1, 100).reduce((acc, px) => {
        acc[`${px}pxr`] = pxToRem(px);
        return acc;
      }, {}),
    }
  }
}

// 사용법
<div className="w-16pxr h-16pxr"} />
```
- pxr을 사용하면 피그마에서의 단위를 사용하더라도 rem이 적용됩니다.

제 생각에는 pxr을 사용하는 것이 제일 좋아보이기는 합니다. 아무래도 시험기간이다 보니, 시험 기간 이후에 진행할 프론트 정기 회의에서 해당 내용을 다루고 전체적으로 적용하는 것은 어떨지 제안 드립니다.

Reply:
**방법 2) tailwind 4 버전으로 업그레이드 하기**

**세 줄 요약**
- 이제는 꼭 4px = xx-1 단위를 사용할 필요가 없어짐. ([참고](https://tailwindcss.com/docs/height#customizing-your-theme))
- tailwind 업그레이드를 하면 새로운 단위를 도입할 필요 없이 spacing 옵션만 바꿔주면 됨.
- 대신 기존에 계산했던 방식은 버려야 함.
기존 h-1 => h-4 (spacing 0.0625rem)

제가 이제 tailwind 4 버전 문서를 읽어봤는데, spacing 관련해서 문서가 변경된 걸 확인했습니다.

예전에는 h-1, h-2 등의 값이 명시가 되어 있었는데, 이제는 spacing에 대한 곱셈 값으로 변경되었네요. 만약 spacing 값을 0.0625rem으로 바꾼다면 이제 h-16 값이 곧 1rem(=16px)을 의미하게 됩니다.

아예 tailwind 버전을 올리고 spacing 값을 변경하는 것도 또다른 방법이 될 수 있을 거 같네요. 이것도 방법 2로 남겨두고, 회의 때 이야기해서 결정하면 좋을 것 같습니다.